app/src/tracing.h

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "constants.h"
+
+#include <time.h>
+#include <regex>
+#include <sstream>
+#include <functional>
+#include <ccf/endpoint_context.h>
+#include <ccf/base_endpoint_registry.h>
+
+namespace scitt
+{
+  std::string REQUEST_ID_HEADER = "x-ms-request-id";
+  std::string CLIENT_REQUEST_ID_HEADER = "x-ms-client-request-id";
```

Comment:
```suggestion
  constexpr std::string_view REQUEST_ID_HEADER = "x-ms-request-id";
  constexpr std::string_view CLIENT_REQUEST_ID_HEADER = "x-ms-client-request-id";
```

`string_view` ensures they aren't mutable.